client/debian/changelog

```diff
@@ -1,3 +1,9 @@
+rust-hwlib (0.9.0~ppa3) UNRELEASED; urgency=medium
```

Comment:
```suggestion
rust-hwlib (0.9.0~ppa3) plucky; urgency=medium
```

Reply:
As a rule of thumb, it's better if changelog/release related changes live in separate commits. Your call though (: